### ISSUE TITLE
Colorizing console output of `disasm` command is slow

### ISSUE THREAD

**Issue by peace-maker** [Gallopsled/pwntools#2239]
The `disasm` command is slow to colorize the output. Coloring is done by disassembling the code three times.

```
$ time disasm 6a095840
   0:    6a 09                    push   0x9
   2:    58                       pop    eax
   3:    40                       inc    eax

real    0m0.867s
user    0m0.137s
sys     0m0.185s

$ time disasm --no-color 6a095840
   0:   6a 09                   push   0x9
   2:   58                      pop    eax
   3:   40                      inc    eax

real    0m0.364s
user    0m0.059s
sys     0m0.114s
```

The code should be disassembled only once.

**Comment by fercevik729** [comment]
I would like to work on this issue.

**Comment by fercevik729** [comment]
Unfortunately I am having some trouble testing it, since I am unable to install binutils on my M2 in accordance with the instructions in pwnlib's documentation. I am getting the same error in issue #2168, even though I've specified the architecture. Also I've tried to test this on Ubuntu 22.04 but I got a `pwnlib.exception.PwnlibException: Could not find 'objdump' installed` 

In both cases `objdump` is already installed but pwnlib can't seem to find it when I execute `disasm`. Could someone lend a hand?
